sampleconfig/configtx.yaml

```diff
@@ -388,6 +388,40 @@ Orderer: &OrdererDefaults
             # SnapshotIntervalSize defines number of bytes per which a snapshot is taken
             SnapshotIntervalSize: 16 MB
 
+    # SmartBFT defines configuration which must be set when the "smartbft"
+    # orderertype is chosen.
+    SmartBFT:
+        # The set of BFT replicas for this network.
+        Consenters:
```

Comment:
I think, this consenters need to be defined at global scope as it is described in the RFC, https://hyperledger.github.io/fabric-rfcs/text/006-bft-based-ordering-service.html#configuration
https://github.com/hyperledger/fabric-protos/blob/main/common/configuration.proto#L35-L49